WalletWasabi/Bases/PeriodicRunner.cs

```diff
@@ -135,14 +113,12 @@ private void LogAndResetLastExceptionIfNotNull()
 			}
 		}
 
-		public async Task StopAsync()
+		public override void Dispose()
 		{
-			Stop?.Cancel();
-			await ForeverTask.ConfigureAwait(false);
-			Stop?.Dispose();
-			Stop = null;
-			Trigger?.Dispose();
-			Trigger = null;
+			TriggeringCts?.Dispose();
+			TriggeringCts = null;
+
```

Comment:
In this file we are logging exceptions using in `UTC` but whatever we write in the logs must be in localtime format, otherwise it is really confusing for the users and for us too.

```c#
Logger.LogInfo($"Exception stopped coming. It came for {(DateTimeOffset.UtcNow - LastExceptionFirstAppeared).TotalSeconds} seconds, {LastExceptionCount} times: {LastException.ToTypeMessageString()}");
```

The problem is in `{LastExceptionCount}`